core/plugins/org.polarsys.capella.core.sirius.analysis/description/common.odesign

```diff
@@ -1389,7 +1399,7 @@
             </initialOperation>
           </ownedTools>
           <ownedTools xsi:type="tool:NodeCreationDescription" documentation="Creates a Entry Pseudo State" name="entry.point" label="%common.msm.entry.point" precondition="aql:self.canCreateEntryExitPoint(containerView)" forceRefresh="true" nodeMappings="//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Mode%20State%20Machine']/@defaultLayer/@nodeMappings[name='MSM_ModeState']" iconPath="/org.polarsys.capella.core.sirius.analysis/icons/full/obj16/Pseudostate_entryPoint.gif">
-            <filters precondition="aql:true"/>
+            <filters precondition="aql:false"/>

```

Comment:
I think the filter hides the tools because the tools are not working properly.
The entry point shall be created under a state and not the first region. 
Can you create a new PR dedicated to the entry/exit points changes as the mapping and tools need to be changed too

Reply:
The filters preconditions were updated in the first patch but not in the second. I don't want to change this behavior because I don't reproduce the tools hidding. I am wrong ?